pkg/core/state/notification_event.go

```diff
@@ -61,7 +61,10 @@ func (aer *AppExecResult) EncodeBinary(w *io.BinWriter) {
 	for _, it := range aer.Stack {
 		stackitem.EncodeBinaryProtected(it, w)
 	}
-	w.WriteArray(aer.Events)
+	w.WriteVarUint(uint64(len(aer.Events)))
```

Comment:
I'm just wondering what's the performance gain this optimisation gives (how much is it?).

Reply:
```
name            old time/op    new time/op    delta
SerializeAER-8    1.14µs ±10%    0.90µs ± 8%  -20.90%  (p=0.000 n=10+10)

name            old alloc/op   new alloc/op   delta
SerializeAER-8      872B ± 0%      800B ± 0%   -8.26%  (p=0.000 n=10+10)

name            old allocs/op  new allocs/op  delta
SerializeAER-8      12.0 ± 0%      10.0 ± 0%  -16.67%  (p=0.000 n=10+10)
```

Reply:
This is for a single aer with one notification event.

Reply:
Why this test is not added to the relevant commit and these results are not mentioned in the commit message?